### ISSUE TITLE
jupyterlab build fails due to a missing file related to `nglview-js-widgets`

### ISSUE THREAD

**Issue by muh-hassani** [nglviewer/nglview#976]
When I try to build jupyterlab (`jupyter lab build`), the build fails due to a missing file related to nglview-js-widgets.
```
[webpack-cli] ModuleNotFoundError: Module not found: Error: Can't resolve './base.css' in '<somepathe>/anaconda3/share/jupyter/lab/staging/node_modules/nglview-js-widgets/style'
```
I tried to remove nglview via conda and then retry to rebuild jupyterlab, but it seems that the error persists.  
I also tried to remove the labextension via `jupyter labextension uninstall nglview-js-widgets`, there I get the error that
```
JupyterLab cannot uninstall nglview-js-widgets since it was installed outside of JupyterLab. Use the same method used to install this extension to uninstall this extension.
```
I appreciate any idea on how I can resolve the problem with the `jupyter lab build`? 
I am using `nglview=3.0.1` from conda-forge.

**Comment by hainm** [comment]
Hi @muh-hassani: 

You don't need to do  those steps. Here is the latest instruction for both notebook and lab:
https://github.com/nglviewer/nglview#released-version

**Comment by hainm** [comment]
Here is a simple script: https://github.com/nglviewer/nglview/blob/master/devtools/nglview-jupyterlab.sh

**Comment by hainm** [comment]
By the way, why do you need `jupyter lab build`?

**Comment by muh-hassani** [comment]
> 
> 
> By the way, why do you need `jupyter lab build`?

Hi @hainm,
thanks for your response. Jupyterlab prompted to include some extensions in the build, when I and other users started a jupyterlab session.
The problem is now solved. I did the following:
```
conda remove nglview 
rm /anaconda3/share/jupyter/lab/extensions/nglview-js-widgets-3.0.1.tgz
jupyter lab clean
jupyter lab build
```
then again reinstalled nglview from conda-forge. Then `jupyter lab build` went well. I don't know exactly what caused the issue honestly, but the above worked in my case.
